client/logic/globals/types.gd

```diff
@@ -92,41 +92,38 @@ func _generate_labels() -> void:
 
 	for unit_key: String in units:
 		var unit: Dictionary = units.get(unit_key)
-		add_text_to_gdscript(file_dumper, unit_key, unit.get("name", "") as String, "Unit name")
+		add_text_to_gdscript(file_dumper, unit_key, unit.get("name", "") as String)
 		add_text_to_gdscript(
 			file_dumper,
 			"%s_DESCRIPTION" % unit_key,
-			unit.get("description", "") as String,
-			"Unit name"
+			unit.get("description", "") as String
```

Comment:
should we add a prefix into this autogenerated IDs? like UNIT_INFANTERY, UNIT_MECH_DESCRIPTION ?

Reply:
Was thinking the same. But what happens, when I change the keys in the base file? Will weblate change it too or drop the keys and create new one?

Reply:
for what I saw before, I think most of them, will match it correctly. but I think will flag them as "needs editing"

Reply:
Since the type represents the translation ID, adding a prefix would require changing all unit IDs and terrain IDs in the types folder, .tscn files, tileset resources, and all map files. It would also lengthen the readable text on the tileset.  I suggest keeping the translation as is, without prefixes. We can still utilize the description field in Weblate.